invisible_cities/io/pmap_io_test.py

```diff
@@ -101,144 +174,149 @@ def test_pmap_writer(config_tmpdir, s12_dataframe_converted, s2si_dataframe_conv
-                s1_t = cpf._time_from_index(s1_indx)
-
-                # S1 and S2
-                s1 = cpf.find_S12(csum, s1_indx, **s1_params._asdict())
-                s2 = cpf.find_S12(csum, s2_indx, **s2_params._asdict())
-                #S2Si
-                sipm = cpf.signal_sipm(sipmrwf[event],
-                                       adc_to_pes_sipm,
-                                       thr=thr.thr_sipm,
-                                       n_MAU=100)
-                SIPM = cpf.select_sipm(sipm)
-                s2si = pf.sipm_s2_dict(SIPM, s2, thr=thr.thr_SIPM)
-
-                # tests:
-                # energy vector and time vector equal in S1 and s2
-                assert len(s1[0][0]) == len(s1[0][1])
-                assert len(s2[0][0]) == len(s2[0][1])
-
-                if s2 and s2si:
-                    for nsipm in s2si[0]:
-                        assert len(s2si[0][nsipm]) == len(s2[0][0])
-
-                # make S1, S2 and S2Si objects (from dicts)
-                S1 = S12(s1)
-                S2 = S12(s2)
-                Si = S2Si(s2si)
-                # store in lists for further testing
-                XS1L.append(s1)
-                XS2L.append(s2)
-                XS2SiL.append(s2si)
-                # produce a fake timestamp (in real like comes from data)
-                timestamp = int(time.time())
-                # write to file
-                write_pmap         (event, S1, S2, Si)
-                write_run_and_event(run_number, event, timestamp)
-
-    # Read back
-    s1df, s2df, s2sidf = read_pmaps(PMAP_file_name)
-    rundf, evtdf = read_run_and_event_from_pmaps_file(PMAP_file_name)
-    # get the dicts
-
-    S1L = df_to_pmaps_dict(s1df)
-    S2L = df_to_pmaps_dict(s2df)
-    S2SiL = df_to_s2si_dict(s2sidf)
-
-    #test
-    for event in range(len(XS1L)):
-        s1 = XS1L[event]
-        if s1: # dictionary not empty
-            s1p = S1L[event]
-            for peak_number in s1p:
-                np.testing.assert_allclose(s1p[peak_number].t,
-                                           s1[peak_number][0])
-                np.testing.assert_allclose(s1p[peak_number].E,
-                                           s1[peak_number][1])
-        s2 = XS2L[event]
-        if s2:
-            s2p = S2L[event]
-            for peak_number in s2p:
-                np.testing.assert_allclose(s2p[peak_number].t,
-                                       s2[peak_number][0])
-                np.testing.assert_allclose(s2p[peak_number].E,
-                                       s2[peak_number][1])
-        s2si = XS2SiL[event]
-        if s2si:
-            sip = S2SiL[event]
-            for peak_number in sip:
-                sipm = sip[peak_number]
-                sipm2 = s2si[peak_number]
-                for nsipm in sipm:
-                    np.testing.assert_allclose(sipm[nsipm], sipm2[nsipm])
+# @mark.slow
+# def test_pmap_electrons_40keV(config_tmpdir):
```

Comment:
Again, if this is useful shouldn't be commented, if not maybe we should remove it. What do you think @jjgomezcadenas ?